### ISSUE TITLE
copy paste typo in initializing batch norm?

### ISSUE THREAD

**Issue by karpathy** [kazuto1011/deeplab-pytorch#44]
https://github.com/kazuto1011/deeplab-pytorch/blob/99312a14a4250dd6fccfe056acf0f201cbf5309d/libs/models/__init__.py#L21

should this be `nn.init.constant_(m.bias, 0)`?

**Comment by kazuto1011** [comment]
Yes definitely! Thank you for reporting. ffbee66 fixed.
